pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -63,4 +64,71 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 			},
 		}, 0),
 	)
+
+	DescribeTable("should accept the WorkloadUpdateMethods", func(oldSpec, newSpec *v1.KubeVirtSpec) {
+		causes := validateWorkloadUpdateMethods(oldSpec, newSpec)
+		Expect(causes).To(BeEmpty())
+	},
+		Entry("should accept LiveMigrate feature gate enabled and workloadUpdateMethod LiveMigrate",
+			newKubeVirtSpec(),
+			newKubeVirtSpec(
+				withFeatureGate(virtconfig.LiveMigrationGate),
+				withWorkloadUpdateMethod(v1.WorkloadUpdateMethodLiveMigrate),
+			),
+		),
+		Entry("should accept LiveMigrate feature gate enabled and no workloadUpdateMethod",
+			newKubeVirtSpec(),
+			newKubeVirtSpec(
+				withFeatureGate(virtconfig.LiveMigrationGate),
+			),
+		),
+		Entry("should accept LiveMigrate feature gate disabled and no workloadUpdateMethod",
+			newKubeVirtSpec(),
+			newKubeVirtSpec(),
+		),
+		Entry("should accept if the misconfiguration was already present",
+			newKubeVirtSpec(
+				withWorkloadUpdateMethod(v1.WorkloadUpdateMethodLiveMigrate),
+			),
+			newKubeVirtSpec(
+				withWorkloadUpdateMethod(v1.WorkloadUpdateMethodLiveMigrate),
+			),
+		),
+	)
+
+	It("should reject LiveMigrate feature gate disabled and workloadUpdateMethod LiveMigrate", func() {
+		causes := validateWorkloadUpdateMethods(
+			newKubeVirtSpec(),
+			newKubeVirtSpec(
+				withWorkloadUpdateMethod(v1.WorkloadUpdateMethodLiveMigrate),
+			))
+		Expect(causes).To(Not(BeEmpty()))
```

Comment:
I could swear I had written this before. Probably forgot to hit the button ...

nit: feel free to ignore; if you `Expect(causes).NotTo(BeEmpty())` you would use less parenthesis. Add I also prefer how `NotTo` reads over `ToNot` .